arbitrator/validator/src/config.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025-2026, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro/blob/master/LICENSE.md
+
+use clap::{Parser, ValueEnum};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Parser)]
+pub struct ServerConfig {
+    /// Host where the server should be run.
+    #[clap(long, default_value = "0.0.0.0")]
+    pub host: String,
+
+    /// Port where the server should be run.
+    #[clap(long, default_value_t = 4141)]
+    pub port: u16,
+
+    /// Logging format configuration.
+    #[clap(long, value_enum, default_value_t = LoggingFormat::Text)]
+    pub logging_format: LoggingFormat,
+}
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default, ValueEnum)]
+pub enum LoggingFormat {
+    #[default]
+    Text,
+    Json,
+}
+
+impl FromStr for LoggingFormat {
```

Comment:
you can add `#[serde(rename_all = "lowercase")]` to LoggingFormat and fully remove `impl FromStr for LoggingFormat`

Reply:
oh, I didn't know that trick; nice one! [217bd80](https://github.com/OffchainLabs/nitro/pull/4173/commits/217bd809b75295b4b7b3faf3e6bad4b09f4e9e96)